web/src/main/java/org/openmrs/web/filter/util/SessionModelUtils.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.web.filter.util;
+
+import org.openmrs.web.filter.initialization.InitializationWizardModel;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpSession;
+import java.lang.reflect.Field;
+import java.util.Enumeration;
+
+/**
+ * Utility class for saving and loading the state of an {@link InitializationWizardModel}
+ * to and from an {@link HttpSession}.
+ */
+public class SessionModelUtils {
+	
+	private static final org.slf4j.Logger log = LoggerFactory.getLogger(SessionModelUtils.class);
+	
+	private static final String PREFIX = "setup.";
+	
+	/**
+	 * Saves all non-null fields of the {@link InitializationWizardModel} to the HTTP session.
+	 * The field name is used as the session key, prefixed with "setup.".
+	 *
+	 * @param session the current {@link HttpSession}, must not be null
+	 * @param model the {@link InitializationWizardModel} to save
+	 */
+	public static void saveToSession(HttpSession session, InitializationWizardModel model) {
+		if (session == null || model == null) return;
```

Comment:
Per our coding conventions, even one-line ifs should have curly braces.

```suggestion
		if (session == null || model == null) {
			return;
		}
```